Classes/Infrastructure/ContentRepository/ConflictsFactory.php

```diff
@@ -130,67 +124,42 @@ private function createConflictFromCommandThatFailedDuringRebase(
+                $eventThatFailedDuringRebase->getEvent()
             ),
             reasonForConflict: $this->createReasonForConflictFromException(
-                $commandThatFailedDuringRebase->exception
+                $eventThatFailedDuringRebase->getException()
             )
         );
     }
 
-    private function extractNodeAggregateIdFromCommand(CommandInterface $command): ?NodeAggregateId
-    {
-        return match (true) {
-            $command instanceof MoveNodeAggregate,
-            $command instanceof SetNodeProperties,
-            $command instanceof SetSerializedNodeProperties,
-            $command instanceof CreateNodeAggregateWithNode,
-            $command instanceof CreateNodeAggregateWithNodeAndSerializedProperties,
-            $command instanceof TagSubtree,
-            $command instanceof DisableNodeAggregate,
-            $command instanceof UntagSubtree,
-            $command instanceof EnableNodeAggregate,
-            $command instanceof RemoveNodeAggregate,
-            $command instanceof ChangeNodeAggregateType,
-            $command instanceof CreateNodeVariant =>
-                $command->nodeAggregateId,
-            $command instanceof SetNodeReferences,
-            $command instanceof SetSerializedNodeReferences =>
-                $command->sourceNodeAggregateId,
-            default => null
-        };
-    }
-
-    private function acquireSubgraphFromCommand(
-        CommandInterface $command,
+    private function acquireSubgraph(
+        EventInterface $event,
         ?NodeAggregateId $nodeAggregateIdForDimensionFallback
     ): ?ContentSubgraphInterface {
         if ($this->workspace === null) {
             return null;
         }
 
-        $dimensionSpacePoint = match (true) {
-            $command instanceof MoveNodeAggregate =>
-                $command->dimensionSpacePoint,
-            $command instanceof SetNodeProperties,
-            $command instanceof SetSerializedNodeProperties,
-            $command instanceof CreateNodeAggregateWithNode,
-            $command instanceof CreateNodeAggregateWithNodeAndSerializedProperties =>
-                $command->originDimensionSpacePoint->toDimensionSpacePoint(),
-            $command instanceof SetNodeReferences,
-            $command instanceof SetSerializedNodeReferences =>
-                $command->sourceOriginDimensionSpacePoint->toDimensionSpacePoint(),
-            $command instanceof TagSubtree,
-            $command instanceof DisableNodeAggregate,
-            $command instanceof UntagSubtree,
-            $command instanceof EnableNodeAggregate,
-            $command instanceof RemoveNodeAggregate =>
-                $command->coveredDimensionSpacePoint,
-            $command instanceof ChangeNodeAggregateType =>
+        $dimensionSpacePoint = match ($event::class) {
+            NodeAggregateWasMoved::class =>
+                $event->succeedingSiblingsForCoverage->toDimensionSpacePointSet()->getFirst(),
```

Comment:
todo this is maybe cheesy? We look into the `succeedingSiblingsForCoverage` ???

Reply:
Nasty.. I created an issue for that https://github.com/neos/neos-development-collection/issues/5360